compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/autofill/ContentType.skiko.kt

```diff
@@ -16,45 +16,50 @@
 
 actual sealed interface ContentType {
     actual companion object {
-        actual val Username: ContentType = throw NotImplementedError()
-        actual val Password: ContentType = throw NotImplementedError()
-        actual val EmailAddress: ContentType = throw NotImplementedError()
-        actual val NewUsername: ContentType = throw NotImplementedError()
-        actual val NewPassword: ContentType = throw NotImplementedError()
-        actual val PostalAddress: ContentType = throw NotImplementedError()
-        actual val PostalCode: ContentType = throw NotImplementedError()
-        actual val CreditCardNumber: ContentType = throw NotImplementedError()
-        actual val CreditCardSecurityCode: ContentType = throw NotImplementedError()
-        actual val CreditCardExpirationDate: ContentType = throw NotImplementedError()
-        actual val CreditCardExpirationMonth: ContentType = throw NotImplementedError()
-        actual val CreditCardExpirationYear: ContentType = throw NotImplementedError()
-        actual val CreditCardExpirationDay: ContentType = throw NotImplementedError()
-        actual val AddressCountry: ContentType = throw NotImplementedError()
-        actual val AddressRegion: ContentType = throw NotImplementedError()
-        actual val AddressLocality: ContentType = throw NotImplementedError()
-        actual val AddressStreet: ContentType = throw NotImplementedError()
-        actual val AddressAuxiliaryDetails: ContentType = throw NotImplementedError()
-        actual val PostalCodeExtended: ContentType = throw NotImplementedError()
-        actual val PersonFullName: ContentType = throw NotImplementedError()
-        actual val PersonFirstName: ContentType = throw NotImplementedError()
-        actual val PersonLastName: ContentType = throw NotImplementedError()
-        actual val PersonMiddleName: ContentType = throw NotImplementedError()
-        actual val PersonMiddleInitial: ContentType = throw NotImplementedError()
-        actual val PersonNamePrefix: ContentType = throw NotImplementedError()
-        actual val PersonNameSuffix: ContentType = throw NotImplementedError()
-        actual val PhoneNumber: ContentType = throw NotImplementedError()
-        actual val PhoneNumberDevice: ContentType = throw NotImplementedError()
-        actual val PhoneCountryCode: ContentType = throw NotImplementedError()
-        actual val PhoneNumberNational: ContentType = throw NotImplementedError()
-        actual val Gender: ContentType = throw NotImplementedError()
-        actual val BirthDateFull: ContentType = throw NotImplementedError()
-        actual val BirthDateDay: ContentType = throw NotImplementedError()
-        actual val BirthDateMonth: ContentType = throw NotImplementedError()
-        actual val BirthDateYear: ContentType = throw NotImplementedError()
-        actual val SmsOtpCode: ContentType = throw NotImplementedError()
+        actual val Username: ContentType = PlatformContentType(-1)
```

Comment:
We should replace -1 by increasing number. Otherwise this comparison won't work correctly:
```
when (type) {
  Password -> 
  EmailAddress ->
  ...
}
```

Reply:
It's not adopted code. `type` is private to file and cannot be used anywhere.

It's in the scope of https://youtrack.jetbrains.com/issue/CMP-7154

Reply:
It's in the scope of https://youtrack.jetbrains.com/issue/CMP-7154

Reply:
Done